web-client/src/app/components/profile-card-horizontal/profile-card-horizontal.component.ts

```diff
@@ -6,9 +6,9 @@ import { Component, Input, OnInit } from '@angular/core';
   styleUrls: ['./profile-card-horizontal.component.scss'],
 })
 export class ProfileCardHorizontalComponent implements OnInit {
-  @Input() title: string | undefined;
-  @Input() fullName!: string;
-  @Input() cellphone!: string;
+  @Input() title?: string;
+  @Input() fullName?: string;
```

Comment:
Did you intend to change the `!` to `?` here as well?

Reply:
Yup: I'm generally in favour of using `?` and checked accessors rather than `!`, to have more predictable crashes and error reporting if the condition ever gets violated..